contracts/clients/tsconfig.json

```diff
@@ -68,5 +68,5 @@
     /* Advanced Options */
     "skipLibCheck": true,                           /* Skip type checking of declaration files. */
     "forceConsistentCasingInFileNames": true        /* Disallow inconsistently-cased references to the same file. */
-  }
+  },
```

Comment:
Interesting trailing comma here. That's invalid json, but I suppose `tsc` is ok with it?

Reply:
Apparently haha, will remove.